taworks/taform/templates/taform/application.html

```diff
@@ -13,12 +13,39 @@
     <form>
         <button class="button" formaction="{% url 'home' %}">Home</button>
     </form>
+    <table class="table2">
+        <tr>
+            <td><b>Application status:</b></td>
+            {% if app_status ==  'Closed' %}
+            <td id="p01">{{app_status}}</td>
+            {% else %}
+            <td class="green_txt">{{app_status}}</td>
+            {% endif %}
+            <td></td>
+        </tr>
+        <tr>
+            <td><b>Last status change:</b></td>
+            <td>{{status_date}}&emsp;</td>
+            <td><form action="{% url 'application' %}" method="POST">
+                {% csrf_token %}
+                <input type="submit" name="app_status" value="Change Application Status"/>
+            </form></td>
+        </tr>
+    </table>
 {% endif %}
```

Comment:
I might be missing something , but is this end if doing anything?

Reply:
The way GitHub is displaying it makes it hard to see, but I believe its the endif for your "If AC" statement

![screen shot 2018-02-05 at 7 08 08 pm](https://user-images.githubusercontent.com/22134649/35835386-663bed92-0aa8-11e8-92be-dcae8ed3d5d6.png)